src/ui/components/navigation/navigationcomponent.js

```diff
@@ -472,8 +473,22 @@ export default class NavigationComponent extends Component {
     }
 
     // We want to persist the params from the existing URL to the new
-    // URLS we create.
+    // URLS we create, except for facets/filters/pagination
     params.set('tabOrder', this._tabOrder);
+
+    params.delete(StorageKeys.SEARCH_OFFSET);
+
+    let prefixes = this.componentManager.getComponentNamesForComponentTypes([
+      'Facets', 'FilterBox', 'FilterOptions', 'RangeFilter', 'DateRangeFilter', 'SortOptions'
```

Comment:
Could we use FacetsComponent.type, FilterOptionsComponent.type, etc. here? That way, if for some reason we change the type name, we don't have to change it here as well.